scripts/device-test.ps1

```diff
@@ -24,7 +24,8 @@ try
 {
     if (!$Tfm)
     {
-        $Tfm = 'net8.0'
+        $Tfm = 'net9.0'
+        #$Tfm = 'net8.0'
```

Comment:
This changes the default TFM from net8.0 to net9.0... what that intentional? Any reason to leave the old one commented out? This is a parameter that gets passed to the script anyway so it's easy to switch between one tfm and the other simply by passing a different parameter.

Reply:
Done